test/e2e/app-launch.e2e.test.js

```diff
@@ -70,7 +70,7 @@ test.describe.serial('Application launch', async () => {
     await mainWindow.click('button:has-text("Continue")')
     await mainWindow.click('button:has-text("Continue")')
     await mainWindow.click('button:has-text("Accept")')
-    expect(await mainWindow.title()).toBe('Saturn')
+    expect(await mainWindow.title()).toBe('Filecoin Station')
```

Comment:
This does not seem like the right fix to me. I think you should change the steps above to click on the button leading to the Saturn page and then keep this `expect()` unchanged.

Reply:
right, the test wants to navigate to Saturn 👍

Reply:
Maybe there should also be a test called "navigate through the onboarding flow"? 

My intention behind these end-to-end tests was to verify that we can render all pages ("Station" and "Saturn" at that time), thus preventing trivial errors.

I think this is still useful, and therefore it would be good to update our end-to-end test suite to keep visiting all pages.

At the same time, I am fine to remove tests and pages that are no longer useful.

Reply:
To expedite the landing of this large PR, I am proposing to keep this test as is but change the name from "navigate to Saturn" to something like "complete the onboarding flow" or a different name matching the actual test implementation.

Reply:
Your fix in [067c2de](https://github.com/filecoin-project/filecoin-station/pull/140/commits/067c2de65341a43f9967761f78e212b325f157ee) is good too 👍🏻